woocommerce/woocommerce-gutenberg-products-block
# Enable auto-expand for product selection in HandPickedProductsControl

This PR adds a new `__experimentalExpandOnFocus` attribute to the component within the `HandPickedProductsControl`. The value is set to `true`, enabling automatic expansion of the list when the user focuses on the product selection field.

This change aims to improve the user experience by making it easier for the user to see all available options when they focus on the selection list.





Fixes #10230


### Testing

1. Create a new post.
2. Insert the 'Product Collection' block.
3. Select the block. In the sidebar settings, add the 'Hand-picked products' filter.
4. Click on the 'Pick some products' input field.
    - Confirm that as soon as you focus on or click into the input field, a list of products appears from which you can select the desired products.

Here is a quick demo:

https://github.com/woocommerce/woocommerce-blocks/assets/16707866/142297e4-2c40-4f41-b268-d466085d2b58



### WooCommerce Visibility





### Changelog

> Product Collection - display list of available products on focus in handpicked products filter
